### DISCUSSION TITLE
Load hybrid-clip in open_clip

### DISCUSSION THREAD

**Discussion by paramtatv** [mlfoundations/open_clip#892] [Q&A]
Is there a way to load a checkpoint from hybrid-clip?
I have trained a model on clip-italian. Now I want to finetune using open_clip for stability reasons.

I tried [this](https://github.com/clip-italian/clip-italian/issues/29), but have issues.

**Comment by rwightman** [comment]
@code-ishwar it can be converted, but keys are different and some layers might be organized a bit differently so needs to be remapped. I'm not aware of any plug and play code going this direction... leaving as discussions in case someone else does
